protobuf/lib/src/protobuf/pb_list.dart

```diff
@@ -23,12 +30,14 @@ class PbList<E> extends ListBase<E> {
         _check = check;
 
   PbList.unmodifiable()
-      : _wrappedList = const [],
+      : _wrappedList = <E>[],
         _check = _checkNotNull,
         _isReadOnly = true;
 
-  PbList.from(List from)
-      : _wrappedList = List<E>.from(from),
+  PbList.from(List<E> from)
+      // Use `<E>[]` to allocate the list, see [_wrappedList] documentation.
+      // ignore: prefer_spread_collections
```

Comment:
It may be that the spread operator gets more optimization than doing follow-up method calls.

Why not make this `_wrappedList = <E>[...from]`?

(similar below